service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -69,4 +69,16 @@ public function isValid(): bool
             ->setData(get_object_vars($this))
             ->isValid();
     }
+
+    public function toArray(): array
+    {
+        return [
+            'personType' => $this->personType,
+            'firstName' => $this->firstName,
+            'lastName' => $this->lastName,
+            'dob' => $this->dob,
+            'address' => $this->address,
+            'lpas' => $this->lpas,
+        ];
+    }
```

Comment:
Not a blocker, probably for a refactor, but we should use `getPersonType()` accessors rather than converting to array. Otherwise we're just setting traps for later where we e.g. rename `'dob'` and don't realise it's in use in the controller.